src/components/Inputs/TextInput/TextInput.tsx

```diff
@@ -55,10 +56,20 @@ const TextInput: React.FC<TextInputProps> = ({
 
 	const handleChange = (event: any) => {
 		const newValue = event.target.value;
-		if (validate(newValue) && onChange) return onChange(event.target.value);
+		if (validate(newValue) && onChange) {
+			return onChange(format(event.target.value));
+		}
+	};
+
+	const format = (str: string) => {
+		if (lowercase) {
+			return str.toLowerCase();
```

Comment:
This doesn't seem to happen when requesting a domain to mint; ie; automatically changing capital letters to lowercase.

Reply:
Currently the 'lowercase' prop isn't set on the input in the Mint flow -- I implemented this functionality, but it made the input feel broken so I opted for an error message instead. Left it in just in case we need it in future design iterations (probably will)